index.md

```diff
@@ -17,139 +43,288 @@ layout: default
+          <p>Orchard</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-02.png" alt=".Net Core">
+          <p>.Net Core</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-03.png" alt="ReactJS">
+          <p>ReactJS</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-04.png" alt="React Native">
+          <p>React Native</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-05.png" alt="C# 6.0">
+          <p>C# 6.0</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-06.png" alt="Cordova">
+          <p>Cordova</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-07.png" alt="Git and Github">
+          <p>Git and Github</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-08.png" alt="HTML5">
+          <p>HTML5</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-09.png" alt="CSS">
+          <p>CSS</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-10.png" alt="Java">
+          <p>Java</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-11.png" alt="Javascript">
+          <p>Javascript</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-12.png" alt="JQuery">
+          <p>JQuery</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-13.png" alt="Objective C">
+          <p>Objective C</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-14.png" alt="Phython">
+          <p>Phython</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-15.png" alt="Selenium">
+          <p>Selenium</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-16.png" alt="Slack">
+          <p>Slack</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-17.png" alt="Trello and Trello API">
+          <p>Trello and Trello API</p>
+        </li>
+        <li>
+          <img src="./assets/images/logo-18.png" alt="Window Azure">
+          <p>Window Azure</p>
+        </li>
+      </ul>
+      <div class="bounce"><i class="fa glyphicon glyphicon-menu-down"></i><i class="fa glyphicon glyphicon-menu-down"></i></div>
     </div>
   </div>
-  <div class="based-area">
-    <div class="area-inner">
-      <div class="container">
-          <h2>We are based in Bangkok</h2>
-          <h3>We've been developing software for over ten years</h3>
-        <ul class="clearfix">
-          <li><a href="#">Test Driver Development</a></li>
-          <li><a href="#">Continuous Integration</a></li>
-          <li><a href="#">End to end testing</a></li>
-          <li><a href="#">Agile Methodologies</a></li>
-          <li><a href="#">Microsoft Gold Parners</a></li>
-        </ul>
-        <p></p>
-      </div>
+</section>
+
+<section class="content-services">
+<div class="based-area">
+<div class="area-inner">
+<div class="container">
```

Comment:
It would be nice if we could have only `<section class="content-services">` and get rid of all the nested `<div>` elements. Why are the divs required? Can the rules defined for those divs be applied to the `content-services` section?